projects/plugins/wpcomsh/changelog/remove-wpcomsh-staging-sites-disable-staging-mode

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: removed
+Comment: The changes are no longer needed after https://github.com/Automattic/jetpack/pull/37023 got merged.
```

Comment:
Could we clarify the wording in case it goes to the changelog? For example:
```suggestion
Comment: Removed code that disables Jetpack staging mode for WordPress.com staging sites.
```

Reply:
Thanks for the suggestion. I updated the changelog in https://github.com/Automattic/jetpack/pull/38355/commits/2bda1cac6cfb4342a46659d69236259e0f0242e0.